### ISSUE TITLE
Odd pull-down list in debugger

### ISSUE THREAD

**Issue by ericwinger** [ericwinger/Jade#76]
Using reproduction case in #75, I see this odd looking pull down list: 

![image](https://user-images.githubusercontent.com/36212516/44357526-1c688a00-a467-11e8-8deb-fcff01067fc0.png)

**Comment by jgfoster** [comment]
This is a list of GsProcess instances that are on the server. You can see more detail if you select one and look at its stack. You an also see them by selecting the "Browse Processes" (Ctrl+R) menu item.

**Comment by ericwinger** [comment]
Shouldn't the "nil" be a bit more descriptive. Perhaps show the stop frame?

**Comment by jgfoster** [comment]
The 'nil' comes from RowanService>>printOn: and refers to the 'name' instance variable. I'm not clear on what 'name' represents and have the impression that it is deprecated. The variable exists in the client and not in the server (along with several other instance variables).

**Comment by ericwinger** [comment]
There is some cleanup remaining in this area. The name is used by all the services - project, package, and class. It represents the name of the underlying object.  The method service's name is it's selector. 

Also, to add to the confusion, the name is held in different classes on the client & server. That is part of the necessary cleanup. 

I wonder if the display could show the print string of the method service representing the top stack of the frame?  (just an idea and I'm open to better ideas)

HTH

**Comment by ericwinger** [comment]
Hmm, not sure this is what was intended after merging #90 

At minimum the frame count is wrong. The code I ran is below the image. 

![image](https://user-images.githubusercontent.com/36212516/44424898-4a210200-a53f-11e8-9e14-ba97cc594f29.png)

```
streamArray2

	| array ws |
	array := Array with: 'abc' with: 'def'.
	self halt. 
	ws := WriteStream on: String new. 
	1 to: array size do:[:idx |
			| blockTemp |
			blockTemp := array at: idx.
			ws nextPutAll: blockTemp.
			ws space.
	].
	^ws contents
```

**Comment by jgfoster** [comment]
Some simple options (< 1 hr):
1. Remove the reference to the number of frames
2. Add '(some may be hidden)' to the frame count
3. Disable the feature that selects a meaningful first line and hides those above (see JadeDebugger>>stackInitialSelectionData and callers) so that the frame count is accurate

More complex options (> 1 hr):
4. Push the stack trim back into the RowanProcessService so that the actual size is correct
5. Remember the number of frames trimmed and adjust the display
6. Use the "top" (non-hidden) frame as the thing to display

Others are possible as well. Let me know how you would like me to proceed.

**Comment by ericwinger** [comment]
How about #1 if it's a quick and dirty fix?

**Comment by jgfoster** [comment]
Fixed by #93.

**Comment by ericwinger** [comment]
Confirmed frame count is removed.
